kernel/kselftest.py.data/ksmtest.yaml

```diff
@@ -0,0 +1,4 @@
+component: !mux
+        test_type: "-M"
+        Size: "-s"
+        Dup_MM_Area: "50"
```

Comment:
I don't think these are only values needed for this test.
So you can leave them as blank
        test_type: " "
        Size: " "
        Dup_MM_Area: " "
        
        make sure you add the expected values for this input parameters in Readme file.
        if possible add some example for expected values.